src/test/java/hudson/plugins/mercurial/MercurialSCMTest.java

```diff
@@ -24,16 +24,34 @@
 
 package hudson.plugins.mercurial;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import hudson.FilePath;
 import hudson.model.Action;
+import hudson.model.TaskListener;
 import hudson.model.Actionable;
-import org.junit.Test;
+import hudson.plugins.mercurial.MercurialSCM.MercurialRevision;
 
 import java.io.IOException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
-import static org.junit.Assert.assertEquals;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
 
 public class MercurialSCMTest {
+	
+	private static final String COMMIT_MESSAGE_INCLUSION_REGEX = "^INCLUDE:.*";
```

Comment:
By the way Jenkins sources, including this plugin, conventionally use four-space indentation rather than tabs. Not a big deal but consistency with the existing convention is preferred.